esp-rtos/src/task/xtensa.rs

```diff
@@ -124,6 +124,7 @@ pub(crate) fn yield_task() {
 
 #[cfg(multi_core)]
 #[esp_hal::handler]
+#[ram]
```

Comment:
Really annoyingly the attribute allowlist in `handler` is sensitive to:
- the exact syntax of the ram attribute
- the order of the attributes, for some arcane reason